log4j-core/src/test/java/org/apache/logging/log4j/core/ShutdownTimeoutConfigurationTest.java

```diff
@@ -16,13 +16,14 @@
  */
 package org.apache.logging.log4j.core;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
```

Comment:
static imports go last. We allow wildcard static imports so this change isn't necessary.